src/views/pages/product.ejs

```diff
@@ -7,10 +7,15 @@
 <body>
     <section class="main-container">
         <%- include('../partials/header') %>
-        <%- include('../partials/gallery')%>
+        <div>
```

Comment:
can this div be removed?

Reply:
This is the same file of the Product detail, in this div should be the include of the information of the product